src/ImageSharp.Drawing/Processing/Drawing/DrawBezierExtensions.cs

```diff
@@ -59,27 +59,27 @@ public static IImageProcessingContext<TPixel> DrawBeziers<TPixel>(this IImagePro
         /// </summary>
         /// <typeparam name="TPixel">The type of the color.</typeparam>
         /// <param name="source">The image this method extends.</param>
+        /// <param name="options">The options.</param>
         /// <param name="color">The color.</param>
         /// <param name="thickness">The thickness.</param>
         /// <param name="points">The points.</param>
-        /// <param name="options">The options.</param>
         /// <returns>The <see cref="Image{TPixel}"/>.</returns>
-        public static IImageProcessingContext<TPixel> DrawBeziers<TPixel>(this IImageProcessingContext<TPixel> source, TPixel color, float thickness, PointF[] points, GraphicsOptions options)
+        public static IImageProcessingContext<TPixel> DrawBeziers<TPixel>(this IImageProcessingContext<TPixel> source, GraphicsOptions options, TPixel color, float thickness, PointF[] points)
             where TPixel : struct, IPixel<TPixel>
-            => source.DrawBeziers(new SolidBrush<TPixel>(color), thickness, points, options);
+            => source.DrawBeziers(options, new SolidBrush<TPixel>(color), thickness, points);
 
         /// <summary>
         /// Draws the provided points as an open Bezier path with the supplied pen
         /// </summary>
         /// <typeparam name="TPixel">The type of the color.</typeparam>
         /// <param name="source">The image this method extends.</param>
+        /// <param name="options">The options.</param>
         /// <param name="pen">The pen.</param>
         /// <param name="points">The points.</param>
-        /// <param name="options">The options.</param>
         /// <returns>The <see cref="Image{TPixel}"/>.</returns>
-        public static IImageProcessingContext<TPixel> DrawBeziers<TPixel>(this IImageProcessingContext<TPixel> source, IPen<TPixel> pen, PointF[] points, GraphicsOptions options)
+        public static IImageProcessingContext<TPixel> DrawBeziers<TPixel>(this IImageProcessingContext<TPixel> source, GraphicsOptions options, IPen<TPixel> pen, PointF[] points)
```

Comment:
we will want to make these arrays `param` properties, this is true for all the extension methods ending with arrays

Reply:
Indeed, using `, params PointF[]` is the whole point of this reordering

Reply:
Haha new I'd missed something 😄 Updated.